### PR TITLE
Openapi headers revised

### PR DESCRIPTION
**Pull request by metacertain**
Top up descriptions of request and response headers in bee API

### CODE REVIEW

**Review by acud — APPROVED** [review]
nice

File: openapi/Swarm.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 
 info:
-  version: 0.5.0
+  version: 0.5.1
```
Comment (Eknir) [inline openapi/Swarm.yaml:4]: Please update to 0.5.2
```suggestion
  version: 0.5.2
```

File: openapi/SwarmDebug.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 openapi: 3.0.3
 info:
-  version: 0.5.0
+  version: 0.5.1
```
Comment (Eknir) [inline openapi/SwarmDebug.yaml:3]:
```suggestion
  version: 0.5.2
```

**Review by Eknir — APPROVED** [review]
We'll merge this for this release. Could you update the version to 0.5.2?

**Comment by Eknir** [comment]
> We'll merge this for this release. Could you update the version to 0.5.2?

My comment is completely invalid. LGTM!

**Pull request merged**